automation/check-npm-version.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+'use strict';
+
+/** Check that semver v1 is greater than or equal to semver v2 */
+function semverGte(v1, v2) {
```

Comment:
Can we get a comment here detailing why we have our own version here (to avoid an `npm install`) so it's not mistakenly removed by anybody in the future please?

Reply:
Done!